### PR TITLE
InstructorCoursesPageUiTest failing on live server #4406

### LINKED ISSUE #4406: InstructorCoursesPageUiTest failing on live server
image

### PR DESCRIPTION
**Pull request by LowWeiLin**
Fixes #4406

### CODE REVIEW

**Comment by LowWeiLin** [comment]
@imouto1994 or @kanghj can you review this?

File: src/test/java/teammates/test/cases/ui/browsertests/InstructorCoursesPageUiTest.java [hunk 1]
```diff
@@ -323,6 +323,7 @@ public void testArchiveAction() throws Exception {
         coursesPage = getCoursesPage();
             
         coursesPage.unarchiveCourse(courseId);
+        coursesPage.waitForAjaxLoadCoursesSuccess();
```
Comment (kanghj) [inline src/test/java/teammates/test/cases/ui/browsertests/InstructorCoursesPageUiTest.java]: If this (waiting after un-archiving a course) is needed, then shouldn't we wait after clicking on archive too?

Comment (LowWeiLin) [inline src/test/java/teammates/test/cases/ui/browsertests/InstructorCoursesPageUiTest.java]: Yes, updated.

Comment (imouto1994) [inline src/test/java/teammates/test/cases/ui/browsertests/InstructorCoursesPageUiTest.java]: Good to me :+1:

**Comment by kanghj** [comment]
Looks good to me. There are other parts of the code like InstructorCourseEditPageUiTest.testDeleteCourseAction where InstructorCoursesPage are used. We should add waits there too, I think. We can either make a new issue for that or fix that here.

**Comment by LowWeiLin** [comment]
Hmm, we do not need to wait for the ajax to complete if it is not being tested.

**Comment by LowWeiLin** [comment]
That test has something wrong though.
```
______TS("delete course then proceed");
InstructorCoursesPage coursePage = courseEditPage.clickDeleteCourseLinkAndConfirm();
coursePage.verifyContains("Add New Course");
```
It should verify that "The course ... has been deleted.", instead of just being back in the coursePage.

**Pull request merged**